content/docs/2.9/scalers/etcd.md

```diff
@@ -0,0 +1,117 @@
++++
+title = "Etcd"
+availability = "v2.9+"
+maintainer = "Huawei Cloud"
+description = "Scale applications based on an etcd key-value pair. By watching an etcd key, a passively received push mode, the scaler can activate applications with lower load usage than frequent pull mode"
+go_file = "etcd_scaler"
++++
+
+### Trigger Specification
+
+This specification describes the `etcd` trigger that scales based on an etcd key-value pair.
+
+```yaml
+  triggers:
+    - type: etcd
+      metadata:
+        endpoints: 172.0.0.1:2379,172.0.0.2:2379,172.0.0.3:2379
+        key: length
+        value: '5.5'
+        activationValue: '0.5'
+        watchProgressNotifyInterval: '600'
+```
+
+**Parameter list:**
+
+- `endpoints` - Etcd servers' endpoints information. It supports multiple endpoints split by a comma character (`,`).
+- `key` - Name of the etcd key used for the scaler client to get/watch the etcd value from etcd servers.
+- `value` - Target relation between the scaled workload and the etcd value. It will be calculated following this formula: relation = (etcd value) / (scaled workload pods). (This value can be a float)
+- `activationValue` - Target value for activating the scaler. Learn more about activation [here](./../concepts/scaling-deployments.md#activating-and-scaling-thresholds).(Default: `0`, Optional, This value can be a float)
+- `watchProgressNotifyInterval` - Set this parameter to the same as `--experimental-watch-progress-notify-interval` parameter of etcd servers. (Default: `600`, units are seconds, Optional)
+
+### Authentication Parameters
+
+You can use `TriggerAuthentication` CRD to configure the authenticate by tls. It is required you should set `tls` to `enable`. If required for your etcd configuration, you may also provide a `ca`, `cert`, `key` and `keyPassword`. `cert` and `key` must be specified together.
+
+**Credential based authentication:**
+
+**TLS:**
+
+- `tls` - To enable SSL auth for Kafka, set this to `enable`. If not set, TLS for Kafka is not used. (Values: `enable`, `disable`, Default: `disable`, Optional)
+- `ca` - Certificate authority file for TLS client authentication. (Optional)
+- `cert` - Certificate for client authentication. (Optional)
+- `key` - Key for client authentication. (Optional)
```

Comment:
`key` parameter is already defined as `Name of the etcd key used for the scaler client to get/watch the etcd value from etcd servers.`

Reply:
I thought about using 'etcdKey' before, but we had discussed that 'key' was more suitable https://github.com/kedacore/keda-docs/pull/914#discussion_r955176841 . Is there a better name to avoid the duplication?

Reply:
f**k, you are right 🤦 TBH, I don't have any good answer for this...
something like `dbKey`? WDYT @kedacore/keda-docs ?

Reply:
When I saw `etcdKey` I didn't notice that `key` was already in use for authentication

Reply:
It is really not easy to notice. What about `watchKey`? It emphasizes the action of watch